common/src/types/v0/message_bus/pool.rs

```diff
@@ -14,7 +14,7 @@ pub struct GetPools {
 
 /// State of the Pool
```

Comment:
State -> Status

Reply:
Good catches thanks, address all of the state/status :+1: